pandas/io/tests/test_excel.py

```diff
@@ -423,7 +424,26 @@ def test_reader_converters(self):
         for path in (xls_path, xlsx_path):
             actual = read_excel(path, 'Sheet1', converters=converters)
             tm.assert_frame_equal(actual, expected)
-
+    
+    def test_reading_all_sheets(self):
+        # Test reading all sheetnames by setting sheetname to None,
+        # Ensure a dict is returned.
```

Comment:
add the PR number (its the issue number when you don't have an issue); so it can be easily referenced.